pkgs/development/libraries/libtickit/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, libtermkey
+, unibilium
+, libtool
+}:
+stdenv.mkDerivation rec {
+  pname = "libtickit";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "leonerd";
+    repo = "libtickit";
+    rev = "v${version}";
+    hash = "sha256-QCrym8g5J1qwsFpU/PB8zZIWdM3YzOySknISSbQE4Sc=";
+  };
+
+  makeFlags = [
+    "PREFIX=$(out)"
+    "LIBTOOL=${lib.getExe libtool}"
+  ];
+
+  nativeBuildInputs = [ pkg-config libtermkey unibilium libtool ];
```

Comment:
I think the dependencies should be moved to buildInputs, except for pkg-config.

LGTM, except for this.

Reply:
Fixed as of 8af8d6707d99db02748b7a873ea61edf43a62c88.